ethers-contract/ethers-contract-abigen/src/lib.rs

```diff
@@ -147,14 +146,20 @@ impl Abigen {
         self
     }
 
-    /// Specify whether or not to format the code using a locally installed copy
-    /// of `rustfmt`.
-    ///
-    /// Note that in case `rustfmt` does not exist or produces an error, the
-    /// unformatted code will be used.
     #[must_use]
+    #[deprecated = "Use format instead"]
+    #[doc(hidden)]
     pub fn rustfmt(mut self, rustfmt: bool) -> Self {
-        self.rustfmt = rustfmt;
+        self.format = rustfmt;
+        self
+    }
```

Comment:
let's just keep this instead of adding/renaming to `format`

Reply:
are you sure? I think it can be misleading because someone might think that it's using the same rustfmt that they use in CI or editor but then the formatting differs

https://github.com/tokio-rs/prost/blob/14ac71ef3300d49d04392e7963d6afc2eaf6f2c4/prost-build/src/lib.rs#L871-L878

Reply:
let's go with Dani's suggestion, agreed